joomla/joomla-cms
# [4.0] Sticky table header

This pr leverages a fixed bug in chrome 91 to allow the setting of position sticky to table headers.

To test rebuild the css `npm run build:css`

On any admin view containing a table eg list of article or list of menu items make sure that you have more rows in the table than the height of your browser.

Scroll down and the toolbar is sticky but the table headers disappear
Apply this PR
Scroll down and the toolbar is sticky and the table headers are sticky

If your browser does not support this then its just ignored and no change will be observed.


### After
![Uploading demo.gif…]()
